Sources/ReactiveMoya/MoyaProvider+Reactive.swift

```diff
@@ -30,7 +30,7 @@ public extension Reactive where Base: MoyaProviderType {
     /// Designated request-making method with progress.
     func requestWithProgress(_ token: Base.Target, callbackQueue: DispatchQueue? = nil) -> SignalProducer<ProgressResponse, MoyaError> {
         let progressBlock: (Signal<ProgressResponse, MoyaError>.Observer) -> (ProgressResponse) -> Void = { observer in
-            return { progress in
+            { progress in
```

Comment:
imho we should keep the `return` keyword when the only statement is a closure. I don't mind removing it in other cases but with closures I find it hard to read.
An other way to deal with it would be to use nested functions instead of closures ?